python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -264,6 +264,8 @@ def run(self, skyInfo, tempExpRefList, imageScalerList, weightList,
             supplementaryData=None):
         """Assemble the coadd.
 
+        Notes
```

Comment:
I don't think this needs to be in a Notes section, it can just be the extended description as-is.

Reply:
@mrawls No, The extended description should be at most a few sentences. We can't allow the extended description to push the parameter and returns descriptions too far down the page. `Notes` is totally appropriate and necessary here.

https://developer.lsst.io/python/numpydoc.html#extended-summary

Reply:
My bad, I didn't catch that the "Notes" was more than the next sentence that followed! Completely agree with @jonathansick, this is fine as-is.